packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -26,9 +26,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     setupFiles: [resolve('config/polyfills.js')],
     setupTestFrameworkScriptFile: setupTestsFile,
-    testMatch: [
-      '<rootDir>/src/**/__tests__/**/*.js?(x)',
-      '<rootDir>/src/**/?(*.)(spec|test).js?(x)',
+    modulePathIgnorePatterns: [
+      '<rootDir>[/\\\\](build|dist|docs|node_modules|scripts)[/\\\\]',
```

Comment:
I guess nobody is watching and running coverage at the same time, ignore this comment